tests/DuckSmokeTest.js

```diff
@@ -0,0 +1,7 @@
+const { test, expect } = require('@playwright/test');
+
+test('Visible logo test', async ({ page }) => {
```

Comment:
Test name should be meaningful and reflect what the test is doing, e.g. 'Checks that duckduckGo page can be opened'